src/callbacks_step/analysis_dg3d.jl

```diff
@@ -4,18 +4,18 @@ function create_cache_analysis(analyzer, mesh::TreeMesh{3},
                                RealT, uEltype)
 
   # pre-allocate buffers
-  u_local = zeros(uEltype,
-                  nvariables(equations), nnodes(analyzer), nnodes(analyzer), nnodes(analyzer))
-  u_tmp1 = similar(u_local,
-                   nvariables(equations), nnodes(analyzer), nnodes(dg), nnodes(dg))
-  u_tmp2 = similar(u_local,
-                   nvariables(equations), nnodes(analyzer), nnodes(analyzer), nnodes(dg))
-  x_local = zeros(RealT,
-                  ndims(equations), nnodes(analyzer), nnodes(analyzer), nnodes(analyzer))
-  x_tmp1 = similar(x_local,
-                   ndims(equations), nnodes(analyzer), nnodes(dg), nnodes(dg))
-  x_tmp2 = similar(x_local,
-                   ndims(equations), nnodes(analyzer), nnodes(analyzer), nnodes(dg))
+  u_local = StrideArray(undef, uEltype,
```

Comment:
I think it would be good to document somewhere why sometimes `StrideArray` is used. I didn't see any comments at a central location (but I might have missed them), but since there seem to be only 6 places where they are used (in analysis_dg[123]d.jl), maybe you can add them there? Just something short like
```suggestion
  # We use `StrideArray` here since this they are used in performance-relevant places and
  # have been benchmarked to be faster than native `Array`s
  u_local = StrideArray(undef, uEltype,
```

Reply:
Done in 8a000f60